packages/insomnia-inso/webpack/webpack.config.development.js

```diff
@@ -9,7 +9,7 @@ module.exports = merge(base, {
   plugins: [
     new webpack.DefinePlugin({
       __DEV__: true,
-      'process.env.DEFAULT_APP_NAME': JSON.stringify('insomnia-app'),
+      'process.env.DEFAULT_APP_NAME': JSON.stringify('insomnia'),
```

Comment:
This should not be changed, it refers to the user data directory on disk, not the package name.
```suggestion
      'process.env.DEFAULT_APP_NAME': JSON.stringify('insomnia-app'),
```

Reply:
3eead72366259b84a03082623378d956db6a1560